htdocs/accountancy/bookkeeping/balance.php

```diff
@@ -321,8 +321,8 @@
 		$root_account_description = $tmparrayforrootaccount['label'];
 		$root_account_number = $tmparrayforrootaccount['account_number'];
 
-		if (empty($accountingaccountstatic->label) && $accountingaccountstatic->id > 0) {
-			$link = '<a href="'.DOL_URL_ROOT.'/accountancy/admin/card.php?action=update&token='.newToken().'&id='.$accountingaccountstatic->id.'">'.img_edit().'</a>';
+		if (empty($tmparrayforrootaccount['label'])) {
```

Comment:
This bugs.
When you are here the accounting account already exists. Label may or may not exists.
If label does not exists and accountingaccountstatic already exists, we must link to update the account no to create.